Sammuel09/Fast-Food-Fast
# #160704357 API endpoints to create menu options (Admin)

#### What does this PR do?
Allows Admin to post menu option

#### Description of Task to be completed?
- Admin can post menu options
- Each request to post menu is validated that it is in the right format
- The route is protected such that only admin can access it
#### How should this be manually tested?
On Postman, visit https://sammie-fast-food-fast.herokuapp.com/api/v1/auth/login
Sign in as admin using this details:
`username: samuel`
`password: samuel`

Copy the token received in the response in the signup process and add it as part of the request headers in this format:
`x-access-token: token`

Then visit https://sammie-fast-food-fast.herokuapp.com/api/v1/menu
Make a POST Request using data in JSON format with the following format:
`name: string`,
`imageurl: string`
`price: string`

N.B: name is the name of the food item

Expected Data to be returned:
`status: string`,
`data: object`,
`message: string`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2196965/stories/160704357
